chain/consensus/mir/config.go

```diff
@@ -0,0 +1,69 @@
+package mir
+
+import (
+	"github.com/filecoin-project/go-address"
+
+	"github.com/filecoin-project/mir/pkg/checkpoint"
+
+	"github.com/filecoin-project/lotus/chain/ipcagent/rpc"
+)
+
+// ---
+
+type BaseConfig struct {
+	ID            address.Address
```

Comment:
Would you mind adding a comment here on what this ID is?

Reply:
Fixed